src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Command;
+
+use Sulu\Bundle\WebsiteBundle\Sitemap\SitemapProviderPoolInterface;
+use Sulu\Component\Webspace\PortalInformation;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Dump sitemaps.
```

Comment:
Looks to me like this command also writes the sitemap to the filesystem. `Dump` could also mean that it is just writing the sitemap to stdout.